src/sql/src/statements/alter.rs

```diff
@@ -76,7 +76,7 @@ pub enum AlterTableOperation {
     DropColumn { name: Ident },
     /// `RENAME <new_table_name>`
     RenameTable { new_table_name: String },
-    /// `MODIFY COLUMN <column_name> SET FULLTEXT [WITH <options>]`
+    /// `MODIFY COLUMN <column_name> [SET | UNSET] FULLTEXT [WITH <options>]`
```

Comment:
Can we rename it to `SetColumnFulltext` and add the  `UnsetColumnFulltext` here?

Reply:
We can refactor related structs in another pr.